<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogonView</class>
 <widget class="QWidget" name="LogonView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogonView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="HKLM_Run">
     <property name="title">
      <string>HKLM\Software\Microsoft\Windows\CurrentVersion\Run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableMR">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Registry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ImagePath</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="HKLM_RunOnce">
     <property name="title">
      <string>HKLM\Software\Microsoft\Windows\CurrentVersion\RunOnce</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableMRO">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Registry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ImagePath</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="HKLM_RunOnceEx">
     <property name="title">
      <string>HKLM\Software\Microsoft\Windows\CurrentVersion\RunOnceEx</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeMR">
        <column>
         <property name="text">
          <string>Registry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Command</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="HKCU_Run">
     <property name="title">
      <string>HKCU\Software\Microsoft\Windows\CurrentVersion\Run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableUR">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Registry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ImagePath</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="HKCU_RunOnce">
     <property name="title">
      <string>HKCU\Software\Microsoft\Windows\CurrentVersion\RunOnce</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableURO">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Regstry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ImagePath</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="HKCU_RunOnceEx">
     <property name="title">
      <string>HKCU\Software\Microsoft\Windows\CurrentVersion\RunOnceEx</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeUR">
        <column>
         <property name="text">
          <string>Registry Entry</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Verified</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Command</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
